### PR TITLE
First version of Transbank PHP SDK Onepay implementation

### PR DESCRIPTION
**Pull request by RodrigoAyala**
This is the first version of the official WooCommerce plugin for Onepay.

We took the [Transbank PHP SDK](https://github.com/TransbankDevelopers/transbank-sdk-php) to implement it on a Wordpress plugin to run inside of WooCommerce.

It includes the README file for development purposes, an installation document, and the plugin itself.

ezgif com-resize

### CODE REVIEW

**Review by eugenioLopezRamos — APPROVED** [review]

**Comment by leosoto** [comment]
@RodrigoAyala I think the instructions should explain a bit when to use TEST or LIVE (and MOCK makes little sense for a plugin, doesn't it?).

I'd say that the TEST mode should be used in the validation process with Transbank. And that once Transbank has approved the e-commerce, it should be switched to LIVE to make real transactions.

**Comment by RodrigoAyala** [comment]
@leosoto You're right, I'll explain more in the document.

About MOCK, what are you suggesting? Today i'm calling `OnepayBase::integrationTypes();` and MOCK is part of the response. (At the beginning we were using a fixed list to show the endpoint selector).

**Comment by leosoto** [comment]
@RodrigoAyala My only suggestion is that the merchant has no use for a "MOCK" option (and it can potentially confuse him). So it shouldn't be there. Whether you filter it out or go back to a fixed list is your call on what is better internally.

**Pull request merged**
